src/components/Information/Information.tsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
```

Comment:
Since we have the new components file, is this one necessary anymore?

Reply:
Removed this file along with Information and Options